### PR TITLE
fixes API guide changes, create-bucket, aliases

### LINKED ISSUE #2861: cloud URLs in examples should keep "https"
When localhost URLs are replaced in cloud API example code, they should have "https" instead of "http".

##### Relevant URLs
- _Provide relevant URLs (the doc page in question, project issues, community threads, etc.)_

### PR DESCRIPTION
**Pull request by jstirnaman**
Closes #2861

_Describe your proposed changes here._

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Tests pass (no build errors)
- [ ] Rebased/mergeable

### CODE REVIEW

File: shared/text/api/v2.0/buckets/oss/create.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/**
```
Comment (jstirnaman) [inline shared/text/api/v2.0/buckets/oss/create.js:1]: This isn't used anywhere yet. Accidental commit. I'll be changing it to use client library.

File: content/influxdb/cloud/organizations/buckets/create-bucket.md [hunk 2]
```diff
@@ -83,3 +83,35 @@ influx bucket create -n my-bucket -o my-org -r 72h
...
+| Requirement          | Include by                                               |
+|:-----------          |:----------                                               |
+| Organization         | Use `orgID` in the JSON payload.                |
+| Bucket               | Use `name` in the JSON payload.                 |
+| Retention Rules      | Use `retentionRules` in the JSON payload.    |
+| Authentication token | Use the `Authorization: Token` header.                   |
+
+#### Example
+
+The URL depends on the version and location of your InfluxDB 2.0 instance _(see [InfluxDB URLs](/{{< latest "influxdb" >}}/reference/urls/))_.
```
Comment (sanderson) [inline content/influxdb/cloud/organizations/buckets/create-bucket.md:110]:
```suggestion
The URL depends on your InfluxDB Cloud region _(see [InfluxDB URLs](/influxdb/cloud/reference/regions/))_.
```

File: content/influxdb/cloud/organizations/buckets/create-bucket.md [hunk 3]
```diff
@@ -83,3 +83,35 @@ influx bucket create -n my-bucket -o my-org -r 72h
...
+
+```sh
+{{% get-shared-text "api/v2.0/buckets/oss/create.sh" %}}
+```
+
+_For information about **InfluxDB API options and response codes**, see
+[InfluxDB API Buckets documentation](/influxdb/v2.0/api/#operation/PostBuckets)._
```
Comment (sanderson) [inline content/influxdb/cloud/organizations/buckets/create-bucket.md:117]:
```suggestion
[InfluxDB API Buckets documentation](/influxdb/cloud/api/#operation/PostBuckets)._      
```

File: content/influxdb/cloud/api-guide/api_intro.md [hunk 4]
```diff
@@ -7,6 +7,9 @@ menu:
   influxdb_cloud:
     name: Quick start
     parent: Develop with the API
+aliases:
+  - /influxdb/v2.0/tools/api/
```
Comment (sanderson) [inline content/influxdb/cloud/api-guide/api_intro.md:11]:
```suggestion
  - /influxdb/cloud/tools/api/
```

File: content/influxdb/v2.0/api-guide/api_intro.md [hunk 5]
```diff
@@ -55,10 +58,16 @@ Include your authentication token as an `Authorization` header in each request.
 
 Postman is another popular tool for exploring APIs. See how to [send authenticated requests with Postman](/{{< latest "influxdb" >}}/api-guide/postman/#send-authenticated-api-requests-with-postman).
 
-## Bucket API
+## Buckets API
 
 Before writing data you'll need to create a Bucket in InfluxDB.
-[Create a bucket](/influxdb/v2.0/buckets/developer-tools/api/) using an HTTP request to the InfluxDB API `bucket` endpoint.
+[Create a bucket](/{{< latest "influxdb" >}}/organizations/buckets/create-bucket/#create-a-bucket-using-the-influxdb-api) using an HTTP request to the InfluxDB API `/buckets` endpoint.
```
Comment (sanderson) [inline content/influxdb/v2.0/api-guide/api_intro.md:64]:
```suggestion
[Create a bucket](/influxdb/v2.0/organizations/buckets/create-bucket/#create-a-bucket-using-the-influxdb-api) using an HTTP request to the InfluxDB API `/buckets` endpoint.
```

**Review by sanderson — CHANGES_REQUESTED** [review]

**Pull request merged**